index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>My portfolio</title>
+  </head>
+  <body>
+    <section class="hero">
+      <header class="nav-container">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav class="nav-bar">
+          <ul class="menu-list">
+            <li class="menu-item"><a href="#">Portfolio</a></li>
+            <li class="menu-item"><a href="#">About</a></li>
+            <li class="menu-item"><a href="#">Contact</a></li>
+          </ul>
+          <div class="mob-menu">
+            <div><img src="./assets/icons/menu.svg" alt="Menu" /></div>
+          </div>
+        </nav>
+      </header>
+      <section class="bio-content">
+        <div class="bio-content-wrapper">
+          <h1>I’m Wilfried. <br />Glad to see you!</h1>
+          <div class="bio">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </div>
```

Comment:
- [ ] Awesome job on your project! **I have a friendly suggestion to improve the HTML structure**. Would you kindly consider changing this div to a p tag? 😁

Reply:
Good idea :+1: 
Thank you, I apply the change